lib/slack-ruby-bot/mvc/controller/base.rb

```diff
@@ -0,0 +1,111 @@
+module SlackRubyBot
+  module MVC
+    module Controller
+      class Base
+        include ActiveSupport::Callbacks
+
+        class << self
+          attr_reader :abstract
+          alias abstract? abstract
+
+          def controllers
+            Base.instance_variable_get(:@controllers)
```

Comment:
Totally. This is by no means a blocker. I [just used](https://github.com/mongoid/mongoid-collection-snapshot/commit/0ab79e97c0d66e38b2be07ef00da67befdb0abd6) `instance_variable_set` too, so I'm caught red-handed.